build/Targets/Settings.props

```diff
@@ -71,6 +71,9 @@
 
     <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\</RoslynNetSdkRootPath> 
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+    <!-- We should be able to remove this extra property once Roslyn projects include the Sdk="Microsoft.NET.Sdk" attribute. -->
```

Comment:
@jaredpar is there a bug tracking? (making Roslyn projects consume SDK using conventional SDK="Microsoft.Targets.Sdk" mechanism)